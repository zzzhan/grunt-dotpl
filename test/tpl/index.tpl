<html>
  <meta charset="utf-8">
  <head><title>${title}</title></head>
  <body>
    <p>${content}</p>
	<ul>
	  <tpl for="menu">
	  <li id="${id}">${name}</li>
	  </tpl>
	</ul>
	<div>${foo}</div>
  </body>
</html>