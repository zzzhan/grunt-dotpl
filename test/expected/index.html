<html>
  <meta charset="utf-8">
  <head><title>grunt-dotpl testing</title></head>
  <body>
    <p>This is test.</p>
	<ul>
	  
	  <li id="1">item1</li>
	  
	  <li id="2">item2</li>
	  
	</ul>
	<div>Generated by grunt-dotpl</div>
  </body>
</html>